stdlib/public/core/Collection.swift

```diff
@@ -584,8 +584,7 @@ extension SequenceType
   // A fast implementation for when you are backed by a contiguous array.
   public func _initializeTo(ptr: UnsafeMutablePointer<Generator.Element>)
     -> UnsafeMutablePointer<Generator.Element> {
-    let s = self._baseAddressIfContiguous
```

Comment:
if let may means nothing changed? here may needs some defensive codes despends on the define and using of _baseAddressIfContiguous